### ISSUE TITLE
Error message at startup

### ISSUE THREAD

**Issue by kl-henry** [spyder-ide/spyder#15265]
### What steps will reproduce the problem?

Starting Spyder 5.0 from a python 3.9 pyenv Environment.
After acknowloging the error window, it spyder seems to work correct.

### Traceback
```python-traceback
/home/heinz/.pyenv/versions/3.9.2/lib/python3.9/site-packages/setuptools/distutils_patch.py:25: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
  warnings.warn(
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.9.2
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Linux 5.3.18-lp152.69-default

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.21.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.0 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
setuptools >=39.0.0           :  49.2.1 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  0.29.22 (OK)
matplotlib >=2.0.0            :  3.3.4 (OK)
numpy >=1.7                   :  1.20.1 (OK)
pandas >=1.1.1                :  1.2.3 (OK)
scipy >=0.17.0                :  1.6.1 (OK)
sympy >=0.7.3                 :  1.7.1 (OK)
```

**Comment by diegstroyer** [comment]
Same error here:
spyder_kernels >=2.0.1,<2.1.0 : 2.0.1 (NOK)

**Comment by stonebig** [comment]
duplicate of https://github.com/spyder-ide/spyder/issues/15091
fixed in Spyder-5.0.1 coming very soon

patch yourself (of the one character typo) if you're in a hurry https://github.com/spyder-ide/spyder/pull/15097/files

**Comment by ccordoba12** [comment]
@kl-henry, thanks for reporting. This is a duplicate of issue #15093 and it'll be fixed in our next version (5.0.1), to be released shortly.

@diegstroyer, as @stonebig said, yours is a duplicate of issue #15091 and it'll also be fixed in 5.0.1
